### PR TITLE
DIS 2.38

### PR DESCRIPTION
**Pull request by spetebroodt**

### CODE REVIEW

File: develop/TOOLS/DIS/Concepts/Overall_concept_of_the_DataMiner_Integration_Studio.md [hunk 1]
```diff
@@ -16,12 +16,23 @@ After installation of the DataMiner Integration Studio extension, Microsoft Visu
...
+
+DIS will set the language version of the C# projects of 4.0, 6.0 or 7.3 depending on the DMA version specified in the *Protocol.Compliancies.MinimumRequired* tag of the protocol XML file:
+
+- If the DMA version is lower than 9.6.11 (or if no *Protocol.Compliancies.MinimumRequired* tag could be found), the language version of the C# projects will be set to 4.0.
+- If the DMA version is equal to or higher than 9.6.11, the language version of the C# projects will be set to one of the following versions:
+
+    - C# 6.0 (when using Visual Studio 2015)
+    - C# 7.3 (when using Visual Studio 2017 or above)
+
+#### [As from DIS v2.38](#tab/tabid-2)
```
Comment (SimonV-Skyline) [inline develop/TOOLS/DIS/Concepts/Overall_concept_of_the_DataMiner_Integration_Studio.md:31]: Did not compile the help to verify but from reading this, it feels to me like "As from DIS v2.38" will be the second tab? (just ignore this comment if I'm miss-interpreting the code here.)
While it should be the first tab and the one active by default.
(we want to, by default, show the current & up to date info. The other tabs are there to have all historical info if needed but should typically not be used that much)

**Review by SimonV-Skyline — CHANGES_REQUESTED** [review]
See my remarks/suggestions/questions.

File: develop/TOOLS/DIS/XML_editor.md [hunk 2]
```diff
@@ -30,7 +30,9 @@ To fix all errors of one particular type or severity at the same time, do the fo
 - To fix all errors of a particular type, right-click an error, and select *Fix \> All errors of this type*.
 - To fix all errors of a particular severity, right-click that severity, and select *Fix \> All errors in this category*.
 
-When, for an error listed in the *DIS Validator* pane, there is more information available than what is being displayed, an information icon will appear next to the description. If you click such an information icon, the additional information will be displayed in a pop-up window.
+When, for an error listed in the *DIS Validator* pane, there is more information available than what is being displayed, an information icon will appear next to the description. If you click such an information icon, the additional information (including possible errors) will be displayed in a pop-up window.
+
+Note that when you try to fix an error with warnings, a popup window will appear to draw your attention to those warnings. Only when you click *OK* in that popup window will the error be fixed.
```
[the remark this answers is not in the document]
Comment (spetebroodt) [reply develop/TOOLS/DIS/XML_editor.md]: will changed to may

File: develop/TOOLS/DIS/Concepts/Overall_concept_of_the_DataMiner_Integration_Studio.md (hunk 1 above)
Comment (spetebroodt) [reply develop/TOOLS/DIS/Concepts/Overall_concept_of_the_DataMiner_Integration_Studio.md]: indeed. well spotted. as from 2.38 will now be first tab.

**Review by SimonV-Skyline — APPROVED** [review]

**Pull request merged**